### PR TITLE
More verbosity when clearing the database & other -cd changes

### PR DESCRIPTION
**Pull request by jagauthier**
Cleaning the database (-cd) can sometimes take a while.  There isn't currently any information to the end user to indicate what is happening.   

This code will report table by table as it deletes and creates them at startup.

Additionally, two other things are changed.
First, if -cd is specified the blacklist is not downloaded. This just saves time.
Secondly, when -cd is complete the process exits.

People run -cd and then stop the process.  This makes -cd more of a utility function intended for one
time run, and not to have the instance running with it enabled.

## Motivation and Context
I'm a big fan of bringing visibility to the users of the application.  I'm also a big fan of implementing code changes that answer questions I see in #help.  

## How Has This Been Tested?
Tested in my local dev environment of a copy of my production database.
```
2017-04-04 13:53:47,038 [        MainThread][           app][    INFO] Blacklist disabled for this session.
2017-04-04 13:53:47,040 [        MainThread][        models][    INFO] Connecting to MySQL database on localhost:3306...
2017-04-04 13:53:47,040 [        MainThread][     runserver][    INFO] Clearing database
2017-04-04 13:53:47,042 [        MainThread][        models][    INFO] Dropping table: Pokemon
2017-04-04 13:53:47,072 [        MainThread][        models][    INFO] Dropping table: Pokestop
2017-04-04 13:53:47,134 [        MainThread][        models][    INFO] Dropping table: Gym
2017-04-04 13:53:47,298 [        MainThread][        models][    INFO] Dropping table: ScannedLocation
2017-04-04 13:53:47,359 [        MainThread][        models][    INFO] Dropping table: Versions
2017-04-04 13:53:47,400 [        MainThread][        models][    INFO] Dropping table: GymDetails
2017-04-04 13:53:47,441 [        MainThread][        models][    INFO] Dropping table: GymMember
2017-04-04 13:53:47,483 [        MainThread][        models][    INFO] Dropping table: GymPokemon
2017-04-04 13:53:47,524 [        MainThread][        models][    INFO] Dropping table: Trainer
2017-04-04 13:53:47,565 [        MainThread][        models][    INFO] Dropping table: MainWorker
2017-04-04 13:53:47,606 [        MainThread][        models][    INFO] Dropping table: WorkerStatus
2017-04-04 13:53:47,688 [        MainThread][        models][    INFO] Dropping table: SpawnPoint
2017-04-04 13:53:47,811 [        MainThread][        models][    INFO] Dropping table: ScanSpawnPoint
2017-04-04 13:53:47,864 [        MainThread][        models][    INFO] Dropping table: SpawnpointDetectionData
2017-04-04 13:53:47,905 [        MainThread][        models][    INFO] Dropping table: LocationAltitude
2017-04-04 13:53:47,946 [        MainThread][        models][    INFO] Dropping table: Token
2017-04-04 13:53:48,436 [        MainThread][        models][    INFO] Creating table: Pokemon
2017-04-04 13:53:49,509 [        MainThread][        models][    INFO] Creating table: Pokestop
2017-04-04 13:53:50,634 [        MainThread][        models][    INFO] Creating table: Gym
2017-04-04 13:53:51,635 [        MainThread][        models][    INFO] Creating table: ScannedLocation
2017-04-04 13:53:52,156 [        MainThread][        models][    INFO] Creating table: GymDetails
2017-04-04 13:53:52,556 [        MainThread][        models][    INFO] Creating table: GymMember
2017-04-04 13:53:53,465 [        MainThread][        models][    INFO] Creating table: GymPokemon
2017-04-04 13:53:53,956 [        MainThread][        models][    INFO] Creating table: Trainer
2017-04-04 13:53:54,272 [        MainThread][        models][    INFO] Creating table: MainWorker
2017-04-04 13:53:54,774 [        MainThread][        models][    INFO] Creating table: WorkerStatus
2017-04-04 13:53:55,990 [        MainThread][        models][    INFO] Creating table: SpawnPoint
2017-04-04 13:53:56,654 [        MainThread][        models][    INFO] Creating table: ScanSpawnPoint
2017-04-04 13:53:57,493 [        MainThread][        models][    INFO] Creating table: SpawnpointDetectionData
2017-04-04 13:53:57,963 [        MainThread][        models][    INFO] Creating table: Token
2017-04-04 13:53:58,524 [        MainThread][        models][    INFO] Creating table: LocationAltitude
2017-04-04 13:53:59,260 [        MainThread][     runserver][    INFO] Drop and recreate is complete. Now remove -cd and restart.
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/models.py [hunk 1]
```diff
@@ -2377,20 +2377,28 @@ def bulk_upsert(cls, data, db):
...
-                      GymMember, GymPokemon, Trainer, MainWorker, WorkerStatus,
-                      SpawnPoint, ScanSpawnPoint, SpawnpointDetectionData,
-                      Token, LocationAltitude], safe=True)
-    db.close()
+    tables = [Pokemon, Pokestop, Gym, ScannedLocation, GymDetails,
+              GymMember, GymPokemon, Trainer, MainWorker, WorkerStatus,
+              SpawnPoint, ScanSpawnPoint, SpawnpointDetectionData,
+              Token, LocationAltitude]
+    for table in tables:
+        log.info("Creating table: %s", table.__name__)
```
Comment (Alderon86) [inline pogom/models.py]: IHMO i think this should be moved to debug output. running this with -v log.txt wont give any information on whats happening.

File: pogom/models.py [hunk 2]
```diff
@@ -2377,20 +2377,28 @@ def bulk_upsert(cls, data, db):
...
+              Token]
     db.connect()
-    db.drop_tables([Pokemon, Pokestop, Gym, ScannedLocation, Versions,
-                    GymDetails, GymMember, GymPokemon, Trainer, MainWorker,
-                    WorkerStatus, SpawnPoint, ScanSpawnPoint,
-                    SpawnpointDetectionData, LocationAltitude,
-                    Token, Versions], safe=True)
+    db.execute_sql('SET FOREIGN_KEY_CHECKS=0;')
+    for table in tables:
+        log.info("Dropping table: %s", table.__name__)
```
Comment (Alderon86) [inline pogom/models.py]: same as above

File: runserver.py [hunk 3]
```diff
@@ -241,6 +244,9 @@ def main():
         elif os.path.isfile(args.db):
             os.remove(args.db)
     create_tables(db)
+    if args.clear_db:
+        log.info("Drop and recreate is complete. Now remove -cd and restart.")
```
Comment (Alderon86) [inline runserver.py]: see above

File: pogom/models.py (hunk 1 above)
Comment (jagauthier) [inline pogom/models.py]: I don't follow you.  This shows up on the console when I run:
```
./runserver.py -cd
./runserver.py -cd -v
```
It shows up on the console and in log.txt when I run
```
./runserver.py -cd -v log.txt
```

**Comment by Alderon86** [comment]
did* of course

File: pogom/models.py (hunk 1 above)
Comment (Alderon86) [inline pogom/models.py]: nvm, im a stupido sometimes

**Comment by FrostTheFox** [comment]
will look at this later

**Pull request merged**